account_vat_period_end_statement/models/account.py

```diff
@@ -723,7 +723,8 @@ def onchange_interest(self, cr, uid, ids, interest, context=None):
     def get_account_interest(self, cr, uid, ids, context=None):
         user = self.pool.get('res.users').browse(cr, uid, uid, context)
         company = user.company_id
-        if company.of_account_end_vat_statement_interest:
+        if company.of_account_end_vat_statement_interest or \
+                self.browse(cr, uid, ids[0], context).interest:
```

Comment:
In generale ```ids``` potrebbe essere più lungo di 1. Quindi il controllo dovrebbe essere fatto su tutti.
Dovresti poter scrivere ad esempio
```
if (
    company.of_account_end_vat_statement_interest or
    any([s.interest for s in self.browse(cr, uid, ids, context)])
):
```